latch_cli/services/init/example_nfcore/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# latch base image + dependencies for latch SDK --- removing these will break the workflow
+from 812206152185.dkr.ecr.us-west-2.amazonaws.com/latch-base-docker:9c8f-main
```

Comment:
Arent you going to add docker into the normal base image?

Reply:
will nuke this after we merge the docker pr